ncm-metaconfig/src/main/metaconfig/ssh/pan/server_config.pan

```diff
@@ -0,0 +1,15 @@
+unique template metaconfig/ssh/server_config;
+
+include 'metaconfig/ssh/schema';
+
+bind "/software/components/metaconfig/services/{/etc/ssh/sshd_config}/contents" = sshd_config_file;
+
+# since final locks the whole path, bind it to a fix value and set it as default too
+#    TODO: support in compiler
```

Comment:
Can we drop the TODO? Doesn't seem to add much information to the reader.

Reply:
sure. out of in terest, do you want this better supported in the compiler, or do you think this overkill?